include/curves/linear_variable.h

```diff
@@ -142,6 +167,7 @@ struct linear_variable : public serialization::Serializable {
   matrix_x_t B_;
   vector_x_t c_;
   bool zero;
+  static const double MARGIN;

```

Comment:
There was already this const definition in Bezier, and now it's required here and in polynomial. Shouldn't we move this in an higher level file to have only one definition for the whole package ?